src/pharmpy/tools/iivsearch/algorithms.py

```diff
@@ -149,3 +153,28 @@ def update_description(model):
     description = create_description(model)
     model = model.replace(description=description)
     return model
+
+
+def _get_eta_from_parameter(model: Model, parameters: List[str]) -> List[str]:
+    # returns list of eta parameters from parameter names
+    iiv_set = set()
+    iiv_names = model.random_variables.names
+    if _has_iiv(model, parameters):
+        for iiv_name in iiv_names:
+            param = get_rv_parameters(model, iiv_name)
+            if set(param).issubset(set(parameters)) and len(param) > 0:
```

Comment:
`issubset` takes any iterable so you don't need to convert `parameters` into a `set()`